client/src/components/SudoModePasswordField/SudoModePasswordField.js

```diff
@@ -145,19 +173,38 @@ function SudoModePasswordField(props) {
     </div>;
   }
 
+  // Prevent a fout on initial render when rendering initiallyCollapsed
+  if (expanded === null) {
+    return null;
+  }
+
   // Render the component
   return <div className="sudo-mode-password-field">
-    <div className="sudo-mode-password-field-inner alert alert-info panel panel--padded">
+    { !expanded && <div className="sudo-mode-password-field__expander alert alert-info panel">
+      <div className="sudo-mode-password-field__expander-text-container">{ verifyMessageValue }</div>
+      <div className="sudo-mode-password-field__expander-button-container">
+        <Button
+          className="sudo-mode-password-field__expander-button font-icon-lock"
+          color="info"
+          onClick={() => handleExpand()}
```

Comment:
```suggestion
          onClick={() => handleConfirmClick()}
```
Would this not work? And make this button render depend on the value of `showVerify`?
i.e `if !showVerify && condensed` where `condensed` replaces `initiallyCollapsed` and no longer has any state associated with it?

Reply:
Updated